src/node.cc

```diff
@@ -1194,6 +1196,8 @@ static void InitializeRecordReplay(int* pargc, char*** pargv) {
       gRecordReplaySaveRecording(nullptr);
     }
 
+    gRecordReplayRememberRecording();
```

Comment:
We're testing the other callbacks are valued before calling. Should we do that here too?

Reply:
No, we shouldn't be doing these tests, they aren't necessary now that we use a fixed version of the recorder library (driver) in each build.